bin/test_integration

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# Run local integration tests (i.e. tests that don't require
+# integration with PCF/Tanzu).
+
+bin_dir="$(dirname $0)"
```

Comment:
nit: I lean towards using `cd $(dirname $0)`, then we don't have to reuse a variable, but that may not be best practice, I've just seen it a lot in our other projects